rollup/dist.config.js

```diff
@@ -13,9 +12,10 @@ export default [
         input: ['./src/index.ts', './src/internal.ts'],
         output: [
             {
-                dir: `${OUTPUT_DIR}`,
-                format: 'cjs',
-                sourcemap: true
```

Comment:
Sourcemaps didn't work for me properly when debugging in vscode. I propose we just don't terse the code, then no special handling is necessary

Reply:
Did you checked chrome's dev-tools? Do you see sources there?